.env

```diff
@@ -77,8 +77,8 @@ REACT_APP_MOCK=true
 REACT_APP_LOCALES="locales"
 
 # IPFS uploading
-REACT_APP_PINATA_API_KEY=
-REACT_APP_PINATA_SECRET_API_KEY=
+REACT_APP_PINATA_API_KEY=foo
+REACT_APP_PINATA_SECRET_API_KEY=bar
```

Comment:
NOOOOOO!!!

Remember to user your local `.env.local`

Then you'll never have this problem again :)

Reply:
:rofl: So that is the solution, will use that one